src/services/logic.service.ts

```diff
@@ -42,15 +41,17 @@ export const handleTransaction = async (req: CustomerCreditTransferInitiation, c
   callback(null, res);
 };
 
-const sendRule = async (rule: Rule, req: CustomerCreditTransferInitiation) => {
-  const toSend = `{"transaction":${JSON.stringify(req)}, "typologies":${JSON.stringify(rule.typologies)}}`;
-
+const sendRule = async (rule: Rule, networkMap: NetworkMap, req: CustomerCreditTransferInitiation) => {
+  const toSend = `{"transaction":${JSON.stringify(req)}, "networkmap":${JSON.stringify(networkMap)}}`;
   let ruleRequest = new FlowFileRequest();
   let objJsonB64 = Buffer.from(JSON.stringify(toSend)).toString("base64");
   ruleRequest.setContent(objJsonB64);
-  ruleEngineService.send(ruleRequest);
+  const ruleService = new RuleEngineService();
+  const ruleClient = ruleService.client(rule.rule_host);
```

Comment:
This is denying the gRPC protocol from being able to perform its magic, allowing the re-use of the same TCP pipe for multiple request, if we're instantiating a new gRPC client for every request. 

We are pulling gRPC out, so it's fine for now.